### PR TITLE
Get information from Firestore for Testimonials and Members

### PR DESCRIPTION
**Pull request by rosariogarcia**

### CODE REVIEW

File: src/app/shared/models/members.model.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+export interface Members {
+    id: string;
```
Comment (dmeneses) [inline src/app/shared/models/members.model.ts]: please use 2 space indentation in all the files

File: src/app/about-us/members/members.service.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { AngularFirestore } from '@angular/fire/firestore';
+
+import { Members } from '../../shared/models/members.model';
+import { DataService } from '../../shared/models/data-service.model';
+import { FirebaseCollections } from '../../shared/models/collections.constant';
+
+@Injectable({
+  providedIn: 'root'
```
Comment (dmeneses) [inline src/app/about-us/members/members.service.ts]: we should remove this as it is already provided by the module.

**Review by dmeneses — CHANGES_REQUESTED** [review]

File: src/app/shared/models/members.model.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    careerEn: string;
+    careerSp: string;
+    displayName: string;
+    email: string;
+    facebook: string;
+    fullname: string;
```
Comment (mauricioarce) [inline src/app/shared/models/members.model.ts]:
```suggestion
    fullName: string;
```

**Review by combimauri — CHANGES_REQUESTED** [review]

**Comment by mauricioarce** [comment]
@rosariogarcia please don't forget to review the observations, I need your code to work on the members

**Comment by rosariogarcia** [comment]
I'm working on this

File: src/app/shared/models/members.model.ts (hunk 3 above)
Comment (rosariogarcia) [reply src/app/shared/models/members.model.ts]: Fixed

File: src/app/about-us/members/members.component.ts [hunk 4]
```diff
@@ -1,42 +1,20 @@
...
-  ];
-  constructor() {}
+  members$: Observable<Members[]>;
 
-  ngOnInit() {}
+  constructor(private membersService: MembersService) {}
+
+  ngOnInit() {
+    this.members$ = this.membersService.getAll();
+
```
Comment (mauricioarce) [inline src/app/about-us/members/members.component.ts]: Remove this extra blank line.

File: src/app/about-us/members/members.service.ts [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+import { Members } from '../../shared/models/members.model';
+import { DataService } from '../../shared/models/data-service.model';
+import { FirebaseCollections } from '../../shared/models/collections.constant';
+
+export class MembersService extends DataService<Members>{
+
+  constructor(http: HttpClient, db: AngularFirestore) {
+    super(http, db, FirebaseCollections.members);
+   }
```
Comment (mauricioarce) [inline src/app/about-us/members/members.service.ts]: Fix indentation here.

File: src/app/home/testimonials/testimonials.component.ts [hunk 6]
```diff
@@ -1,15 +1,19 @@
 import { Component, OnInit } from '@angular/core';
+import { Testimonials } from '../../shared/models/testimonials.model';
+import { TestimonialsService } from './testimonials.service';
+import { Observable } from 'rxjs';
```
Comment (mauricioarce) [inline src/app/home/testimonials/testimonials.component.ts]: Sort your imports (angular imports, libraries imports, local imports) separating them by a blank line.

File: src/app/home/testimonials/testimonials.component.ts [hunk 7]
```diff
@@ -1,15 +1,19 @@
...
 
 @Component({
   selector: 'gp-testimonials',
   templateUrl: './testimonials.component.html',
   styleUrls: ['./testimonials.component.scss']
 })
 export class TestimonialsComponent implements OnInit {
-
-  constructor() { }
+  testimonials$: Observable<Testimonials[]>;
```
Comment (mauricioarce) [inline src/app/home/testimonials/testimonials.component.ts]: Add a blank line after this.

File: src/app/home/testimonials/testimonials.component.ts [hunk 8]
```diff
@@ -1,15 +1,19 @@
...
+  constructor( private testimonialService: TestimonialsService) { }
```
Comment (mauricioarce) [inline src/app/home/testimonials/testimonials.component.ts]:
```suggestion
  constructor(private testimonialService: TestimonialsService) { }
```

File: src/app/home/testimonials/testimonials.service.spec.ts [hunk 9]
```diff
@@ -0,0 +1,25 @@
+import { TestBed } from '@angular/core/testing';
+import { HttpClientModule } from '@angular/common/http';
+import { AngularFirestore } from '@angular/fire/firestore';
+
+import { TestimonialsService } from './testimonials.service';
```
Comment (mauricioarce) [inline src/app/home/testimonials/testimonials.service.spec.ts]: Add blank line after this.

File: src/app/about-us/members/members.component.html [hunk 10]
```diff
@@ -5,7 +5,7 @@ <h4 class="center-align">
   <div class="col s12 m6 member-card valign-wrapper" *ngFor="let member of members">
     <img [src]="member.image" class="member-sub-card">
     <div class="member-sub-card member-info">
-      <p>{{ member.name }}</p>
+      <p>{{ member.fullname }}</p>
```
Comment (mauricioarce) [inline src/app/about-us/members/members.component.html]:
```suggestion
      <p>{{ member.fullName }}</p>
```

**Review by combimauri — CHANGES_REQUESTED** [review]
In addition: Model names should be in singular (Member, Testimonial), I know we already have other models in plural, this is something that should be changed.

**Comment by mauricioarce** [comment]
@rosariogarcia please review.

File: src/app/shared/models/members.model.ts (hunk 1 above)
Comment (rosariogarcia) [reply src/app/shared/models/members.model.ts]: Fixed

File: src/app/about-us/members/members.service.ts (hunk 2 above)
Comment (rosariogarcia) [reply src/app/about-us/members/members.service.ts]: Fixed

**Comment by rosariogarcia** [comment]
I'm working on new comments

File: src/app/about-us/members/members.component.ts (hunk 4 above)
Comment (rosariogarcia) [reply src/app/about-us/members/members.component.ts]: Fixed

File: src/app/about-us/members/members.service.ts (hunk 5 above)
Comment (rosariogarcia) [reply src/app/about-us/members/members.service.ts]: Fixed

**Review by combimauri — APPROVED** [review]

**Pull request merged**